src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,118 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+
+import duke.ui.Ui;
+
+
+public class Task implements Comparable<Task> {
+    protected boolean isDone;
+    protected String content;
+    protected LocalDateTime date;
```

Comment:
Should these be protected? I think we can get away with private as we would not want other classes in this package to be able to access this right?